Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift

```diff
@@ -0,0 +1,551 @@
+import CTensorFlow
+
+/// The `TF_Tensor *` type.
+typealias CTFTensor = OpaquePointer
+
+public class LazyTensor : _AnyTensorHandle {
+    enum Handle {
+        /// Bool indicates if this concrete TFETensorhandle was a result of
+        /// materialization.
+       case conc(TFETensorHandle, Bool)
+        /// Bool indicates whether this is held by a swift variable,
+        /// i.e., is this a live tensor?
+       case sym(LazyTensorOperation, Int, Bool)
+    }
+
+    let handle: Handle
+
+    public var _tfeTensorHandle: TFETensorHandle {
+        switch (handle) {
+        case Handle.conc(let h, _):
```

Comment:
Omit type name in case patterns.
```suggestion
        case concrete(let h, _):
```

Reply:
Done.